docs/user_guide/environment_variables.rst

```diff
@@ -25,6 +25,10 @@ Environment variables allow to overwrite some values of a module configuration f
 - **MTF_REMOTE_REPOS=yes** disables downloading of Koji packages and creating a local repo, and speeds up test execution.
 - **MTF_DISABLE_MODULE=yes** disables module handling to use nonmodular test mode (see `multihost tests`_ as an example).
 - **DOCKERFILE="<path_to_dockerfile"** overwrites the location of a Dockerfile.
+- **OPENSHIFT_LOCAL** enables installing ``origin`` and ``origin-clients`` on local machine
```

Comment:
would be nice to follow the previous definitions: e.g. what is the sample value for this variable?